### PR TITLE
Certificate pinning blog

### PR DESCRIPTION
**Pull request by radhakrishnan-wavemaker**
Changes
----------

* Added certificate pinning blog

please review the changes

### CODE REVIEW

**Review by swethakundaram — APPROVED** [review]

File: website/blog/2020-12-15-certificate-pinning.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+In HTTPS handshake, the server responds with a public certificate issued by Certificate Authority (CA) to establish a connection. If CA or Root certificate was compromised and issues a certificate without the domain owner's consent, the client may face the man-in-the-middle (MTM) attack. To protect users from this vulnerability, an app can employ SSL pinning.
+
+![MITM](/learn/assets/blog_certificate_pinning/mitmdiagram.png)
+
+Certificate Pinning is an additional layer of security which protects communication between client and server. The Standard HTTPS verifies whether the connection is secure but it cannot verify whether you are communicating with the actual server or an intercepted server.
+
+## What Certificate Pinning offers
+
+Standard HTTPS checks only whether the connection is secure. Along with the HTTPS, we use certificate pinning which will verify whether you are communicating with the actual server.
```
Comment (prasanth-reddy-wavemaker-com) [inline website/blog/2020-12-15-certificate-pinning.md:20]: this is incorrect. HTTPS also checks the certificate

File: website/blog/2020-12-15-certificate-pinning.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+* If a trusted certificate authority gets compromised due to security [vulnerability](https://en.wikipedia.org/wiki/Certificate_authority#CA_compromise), the application will not get affected.
+
+## Problem Associated with Certificate Pinning
+
+A security researcher has [warned](https://scotthelme.co.uk/im-giving-up-on-hpkp/) the risk associated with using HTTP Certificate Pinning.
+
+* If the key was accidentally deleted, stolen, hacked, you may face serious application [downtime issues](https://www.smashingmagazine.com/be-afraid-of-public-key-pinning/).
+* Domain hijacking - on Domain-hijacking, you can lose control of certificates, and hijackers can mishandle your certificates.
+
+## What next?
```
Comment (prasanth-reddy-wavemaker-com) [inline website/blog/2020-12-15-certificate-pinning.md:46]: I think we should add that very few sites actually use certificate pinning and one needs to be very careful in evaluating using this feature. After careful consideration, at this point, we have not yet implemented support for certificate pinning in applications build using WaveMaker.

**Review by prasanth-reddy-wavemaker-com — CHANGES_REQUESTED** [review]

File: website/blog/2020-12-15-certificate-pinning.md (hunk 1 above)
Comment (radhakrishnan-wavemaker) [reply website/blog/2020-12-15-certificate-pinning.md]: Made changes to the sentences, Please review again.

File: website/blog/2020-12-15-certificate-pinning.md (hunk 2 above)
Comment (radhakrishnan-wavemaker) [reply website/blog/2020-12-15-certificate-pinning.md:48]: Added the suggested lines under "Problem Associated with Certificate Pinning", Please review again.

**Comment by swethakundaram** [comment]
@prasanth-reddy-wavemaker-com, can you review this again?

**Review by prasanth-reddy-wavemaker-com — APPROVED** [review]
LGTM

**Pull request merged**